.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce behavior
```

Comment:
What about adding a heading for including the Bourbon version being used? Maybe dev/project environment, as well?